dockerfile/cuda12.4.dockerfile

```diff
@@ -115,6 +115,23 @@ RUN cd /tmp && \
     mv amd-blis /opt/AMD && \
     rm -rf aocl-blis-linux-aocc-4.0.tar.gz
 
+# Install NCCL 2.23.4
+RUN cd /tmp && \
+    git clone -b v2.23.4-1 https://github.com/NVIDIA/nccl.git && \
+    cd nccl && \
+    make -j src.build && \
```

Comment:
`make -j ${NUM_MAKE_JOBS} src.build` to avoid OOM?

Reply:
thanks, updated